functional/test_check.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
These tests will run first and will not change the existing secrets, therefore subsequent tasks will execute successfully